dispatch-service/src/main/java/de/muenchen/oss/swim/dispatcher/application/usecase/DispatcherUserCase.java

```diff
@@ -207,7 +215,9 @@ protected void processProtocolFile(final UseCase useCase, final File file) {
             // tag protocol as processed
             fileSystemOutPort.tagFile(file.bucket(), file.path(), Map.of(
```

Comment:
I noticed: a csv that does not match the files on s3 will still be transferred to `_finished` and tagged as `finished`. It cannot be seen on the file itself that there was a mismatch - this is only visible through mail.

Should we maybe annotate that on a tag in the csv-file as well?

Reply:
> Should we maybe annotate that on a tag in the csv-file as well?

What you think about `CSV_Match` to `true/false`?